### ISSUE TITLE
Feature importance using Permutation

### ISSUE THREAD

**Issue by npatta01** [microsoft/interpret#26]
Hi,
At work, we use the "Permutation Importance" method to inspect feature importance.
We use the awesome library [eli5](https://eli5.readthedocs.io/en/latest/autodocs/permutation_importance.html) for that.

Would it be possible to include a version of that in this library?

**Comment by Sandy4321** [comment]
good idea, eli5 is great

**Comment by interpret-ml** [comment]
Thanks for bringing this up! Permutation Importance is definitely a method we've been thinking about. The main challenge is finding the right way to cleanly integrate it into our API. We understand the demand for this feature though, so we'll keep this issue open until we make progress on this.

**Comment by imatiach-msft** [comment]
@npatta01 @Sandy4321 permutation feature importance is available in the interpret-community package here, which is an extension to interpret:
https://github.com/interpretml/interpret-community/blob/master/python/interpret_community/permutation/permutation_importance.py
